opal/managers.py

```diff
@@ -150,3 +150,71 @@ def serialised_active(self, user, **kw):
         episodes = self.filter(**filters)
         as_dict = self.serialised(user, episodes)
         return as_dict
+
+
+class LookupListQueryset(models.QuerySet):
+    def get_content_type(self):
```

Comment:
I'm not really sure what this is adding?
it's marginally shorter when you call it once below, but at the cost of another method?